packaging/templates/puppynet/archive/config.toml

```diff
@@ -90,10 +90,10 @@ gcmode = "archive"
     [jsonrpc.ws]
         enabled = true
         port = 8546
-        # prefix = ""
-        # host = "localhost"
-        # api = ["web3", "net"]
+        host = "0.0.0.0"
+        api = ["eth", "net", "web3", "txpool", "bor"]
```

Comment:
we are not adding debug ? not required ?

Reply:
No, We should not enable debug API by default